src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -289,6 +290,11 @@ static string GenerateTestTimedOutMessage(TimeSpan maxTime)
             {
                 throw new Exception("Executing given and whens took longer than 2 minutes");
             }
+
+            if (completedTask.IsFaulted && completedTask.Exception != null)
```

Comment:
compiler warns about a potential NullRef exception, not sure if a Task can become faulted without an Exception?